data/src/Cardano/Data/StrictMaybe.hs

```diff
@@ -0,0 +1,101 @@
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DeriveTraversable #-}
+
+{- | Strict version of the 'Maybe' type..
+-}
+module Cardano.Data.StrictMaybe
+  ( StrictMaybe (SNothing, SJust),
+
+    -- * Conversion: StrictMaybe <--> Maybe
+    strictMaybeToMaybe,
+    maybeToStrictMaybe,
+
+    -- * accessing the optional component
+    fromSMaybe,
+  )
+  where
+
+import Data.Aeson (FromJSON (..), ToJSON (..))
+import NoThunks.Class (NoThunks (..))
+import GHC.Generics (Generic)
+import Control.DeepSeq (NFData)
+import Cardano.Binary
+  ( FromCBOR (fromCBOR),
+    ToCBOR (toCBOR),
+    decodeBreakOr,
+    decodeListLenOrIndef,
+    encodeListLen,
+  )
+import Data.Default.Class (Default (..))
```

Comment:
Just an FYI: this is formatted by ormolu, but the other modules' export/import lists are formatted by stylish-haskell.